src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,202 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size = 0;
+    private Node<T> head;
+    private Node<T> tail;
+
     @Override
     public void add(T value) {
+        Node<T> newNode = new Node<>(null, value, null);
+        if (size == 0) {
+            head = newNode;
+            tail = newNode;
+            size++;
+            return;
+        }
+        tail.next = newNode;
+        newNode.prev = tail;
+        tail = newNode;
```

Comment:
```suggestion
        Node<T> newNode = new Node<>(tail, value, null);
        if (size == 0) {
            head = newNode;
        } else {
            tail.next = newNode;
        }
        tail = newNode;
```
you can simplify  it